api/v1alpha1/helmchartproxy_types.go

```diff
@@ -72,6 +72,9 @@ type HelmChartProxySpec struct {
 	// Credentials is a reference to an object containing the OCI credentials. If it is not specified, no credentials will be used.
 	// +optional
 	Credentials *Credentials `json:"credentials,omitempty"`
+
+	// TLSConfig contains the TLS configuration for the HelmChartProxy.
```

Comment:
```suggestion
	// TLSConfig contains the TLS configuration for a HelmChartProxy.
	// +optional
```

Should this be optional if it's omitted when empty?

Reply:
Yes, using both `+optional` and `omitempty` is idiomatic - see https://github.com/kubernetes/community/blob/1114fe76ba5de7d38da786e900fc9b715d76441f/contributors/devel/sig-architecture/api-conventions.md#optional-vs-required

Reply:
Alright in that case can you set TLSConfig to be `+optional`?

Reply:
I think we should be good to merge after that!

Reply:
Oops sorry I thought I had!